my_project/tab_sun/charts_sun.py

```diff
@@ -130,7 +131,7 @@ def polar_graph(df, meta, global_local, var):
         marker_size = 3
     else:
         vals = solpos[var]
-        marker_size = (((vals - vals.min()) / vals.max()) + 1) * 4
+        marker_size = ((vals - vals.min()) / (vals.max()-vals.min()) + 1) * 4
```

Comment:
I changed the definition for the 'maker size'; so that can avoid the too-big maker when the unit system is changed to 'IP'.